src/LLMProviders/chatModelManager.ts

```diff
@@ -261,9 +246,7 @@ export default class ChatModelManager {
 
     const modelConfig = this.getModelConfig(model);
 
-    // MUST update it since chatModelManager is a singleton.
-    this.getLangChainParams().modelKey = modelKey;
-    new Notice(`Setting model: ${modelConfig.modelName}`);
```

Comment:
I am removing this notice because buildModelMap is called on any settings value change, which can be annoying for it to show when unrelated settings values are set